src/app/locale/translations/en.json

```diff
@@ -185,6 +185,7 @@
             "chasepay_edit_card": "Edit card",
             "chasepay_logout": "Log out Chase Pay",
             "chasepay_fail_load": "Chase Pay failed to load",
+            "checkoutcom_sepa_mandate_disclaimer":"By accepting this mandate form, you authorize {storeUrl} and Checkout.com, our payment service provider, to send instructions to your bank to debit your account, and your bank to debit your account in accordance with those instructions. You are entitled to a refund from your bank under the terms and conditions of your agreement with your bank. A refund must be claimed within 8 weeks starting from the date on which your account was debited.",
```

Comment:
might it be better to use storeName rather than storeUrl ?

Reply:
In the ticket is specified that string template, and also `payment.stripe_sepa_mandate_disclaimer` has a similar structure, let me ask I'll fix it if it's necessary

Reply:
I just updated the PR following your suggestion
Thanks C: